databases/grade_log/grade_log.rb

```diff
@@ -58,12 +60,20 @@ def question_user(response)
 	end
 end
 
-# driver code for users adding data
-
-puts "Welcome to your gradebook!"
-puts "Do you want to (1) enter grades OR (2) view grades? Please enter a number:"
-response = gets.chomp
-question_user(response)
+#Driver Code
+
+complete = false
+until complete
+	puts "Welcome to your gradebook!"
+	puts "Do you want to (1) enter grades OR (2) view grades? Please enter a number:"
+	response = gets.chomp
+	question_user(response)
+	puts "Are you finished? (y/n)"
+	response_2 = gets.chomp
+	if response_2 == "y"
+		complete = true
+	end
+end
 
```

Comment:
@elizabethhart This is a nice little grading app!

I like that you kept it simple and worked on refining a single table.  A lot of folks tend to runaway with project and get in a little over their heads by adding too much complexity to start.

If you're thinking of refactoring I can give you some stretch goals but if you try them make sure the tables are working before doing your UI code.  Just test things with standard driver code at first.

```
Three tables:

Student
Grade
Class

a student has many grades
a student has many classes through grades

grade belongs to student
grade belongs to classes

classes has many grades
classes has many students through grades
```

This is a bit advanced so let me know if you have questions if you want to work on it.